src/screens/login/SelectInstanceHelp.tsx

```diff
@@ -29,12 +34,24 @@ import MainLayout from 'layouts/MainLayout';
 import Text from 'components/DefaultText';
 import Icon from 'components/DefaultIcon';
 import withTheme, {WithTheme} from 'lib/hoc/withTheme';
-
+import {Button} from 'native-base';
```

Comment:
Can you use our Button component (`import Button from 'components/DefaultButton';`)  and modify it